src/components/vacation-requests-table/vacation-requests-table.tsx

```diff
@@ -172,7 +176,7 @@ const VacationRequestsTable = ({
    */
   const CustomSkeletonTableRows = () => (
     <SkeletonTableRows
-      dataGridHeight={dataGridHeight}
+      dataGridHeight={15}
       dataGridRowHeight={dataGridRowHeight}
```

Comment:
```suggestion
      dataGridHeight={dataGridHeight}
```

Reply:
Also in skeleton-table-rows   
const rowCount = Math.floor((dataGridHeight - dataGridColumnHeaderHeight) / dataGridRowHeight);